src/main/java/com/alevel/backend/controller/UserController.java

```diff
@@ -29,90 +26,92 @@ public UserController(UserService userService) {
             System.out.println("가입 가능한 이메일: " + email);
         } catch (Exception e) {
-            return new ResponseEntity(new DefaultResponse(StatusCode.CONFLICT, ResponseMessage.DUPLICATED, email), HttpStatus.CONFLICT);
+            return ResultResponse.fail(StatusCode.CONFLICT, ResponseMessage.DUPLICATED_EMAIL);
         }
 
         int key = mailService.createKey();
         String subject = "[한잔할래] 회원가입 인증 메일입니다.";
         String content = "인증번호 : " + key +
                 "<br>";
         if (mailService.sendMail(email, subject, content)) {
-            return new ResponseEntity(new DefaultResponse(StatusCode.OK, ResponseMessage.SUCCESS, key), HttpStatus.OK);
+            return ResultResponse.success();
         } else {
-            return new ResponseEntity(new DefaultResponse(StatusCode.BAD_REQUEST, ResponseMessage.FAIL, key), HttpStatus.BAD_REQUEST);
+            return ResultResponse.fail(StatusCode.INTERNAL_SERVER_ERROR, ResponseMessage.INTERNAL_SERVER_ERROR);
         }
     }
 
     /**
      * 회원탈퇴
      */
-    @PutMapping(value = "/user/withdrawal")
-    public ResponseEntity<String> withdrawal(Long id) {
-        return new ResponseEntity(userService.remove(id), HttpStatus.OK);
+    @PutMapping(value = "/users/withdrawal")
+    public ResultResponse withdrawal(Long id) {
+        userService.remove(id);
+        return ResultResponse.success();
     }
 
     /**
      * 별명 변경
      */
-    @PutMapping(value = "/user/username/{id}")
-    public ResponseEntity<String> putUsername(@PathVariable("id") Long id, String username) {
-        try {
-            userService.validateDuplicateUsername(username);
-            userService.updateUsername(id, username);
-            return new ResponseEntity(new DefaultResponse(StatusCode.OK, ResponseMessage.SUCCESS, username), HttpStatus.OK);
-        } catch (Exception e) {
-            return new ResponseEntity(new DefaultResponse(StatusCode.CONFLICT, ResponseMessage.DUPLICATED, username), HttpStatus.CONFLICT);
-        }
+    @PutMapping(value = "/users/username/{id}")
+    public ResultResponse putUsername(@PathVariable("id") Long id, String username) {
+        userService.validateDuplicateUsername(username);
+        userService.updateUsername(id, username);
+        return ResultResponse.success();
     }
 
     /**
      * 비밀번호 변경
      */
-    @PutMapping(value = "/user/password/{id}")
-    public ResponseEntity<String> updatePassword(@PathVariable("id") Long id, String password, String passwordConfirm) {
+    @PutMapping(value = "/users/password/{id}")
+    public ResultResponse updatePassword(@PathVariable("id") Long id, String password, String passwordConfirm) {
         if (!password.equals(passwordConfirm)) {
-            return new ResponseEntity(new DefaultResponse(StatusCode.BAD_REQUEST, ResponseMessage.FAIL), HttpStatus.BAD_REQUEST);
+            return ResultResponse.fail(StatusCode.BAD_REQUEST, ResponseMessage.INVALIDATED_PASSWORD);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)